dmcontent/__init__.py

```diff
@@ -2,4 +2,4 @@
 from .errors import ContentTemplateError, QuestionNotFoundError
 from .questions import ContentQuestion
 
-__version__ = '7.16.0'
+__version__ = '7.18.0'
```

Comment:
Did we skip `7.17`?

Reply:
I've pre-empted the checkboxes PR going in first as 7.17

Reply:
😹 I thought I'd approved that one already - will do so now!